gitrevise/tui.py

```diff
@@ -138,6 +148,9 @@ def noninteractive(
 ) -> None:
     assert head.target is not None
 
+    if args.root:
+        raise ValueError("Incompatible option: --root requires --interactive")
```

Comment:
`--autosquash` would also be allowed here, so perhaps we should mention it in the error as well

Reply:
Updated the error message + docs